docs/concepts/ClusterResourcePlacement/README.md

```diff
@@ -0,0 +1,222 @@
+# ClusterResourcePlacement
+
+## Overview
+
+`ClusterResourcePlacement` concept is used to dynamically select cluster scoped resources (especially namespaces and all 
+objects within it) and control how they are propagated to all or a subset of the member clusters.
+A `ClusterResourcePlacement` mainly consists of three parts:
+- **Resource selection**: select which cluster-scoped Kubernetes
+resource objects need to be propagated from the hub cluster to selected member clusters. 
+  
+  It supports the following forms of resource selection:
+  - Select resources by specifying just the <group, version, kind>. This selection propagates all resources with matching <group, version, kind>. 
+  - Select resources by specifying the <group, version, kind> and name. This selection propagates only one resource that matches the <group, version, kind> and name. 
+  - Select resources by specifying the <group, version, kind> and a set of labels using ClusterResourcePlacement -> LabelSelector. 
+This selection propagates all resources that match the <group, version, kind> and label specified.
+
+  **Note:** When a namespace is selected, all the namespace-scoped objects under this namespace are propagated to the 
+selected member clusters along with this namespace.
+
+- **Placement policy**: limit propagation of selected resources to a specific subset of member clusters.
+  The following types of target cluster selection are supported:
+    - **PickAll (Default)**: select any member clusters with optional matching cluster `Affinity` scheduling rules. 

```

Comment:
May want to reword this and say that it will apply to all member clusters if optional Affinity is not specified.